src/vs/vscode.d.ts

```diff
@@ -1534,12 +1534,31 @@ declare module 'vscode' {
 		 */
 		ignoreFocusOut?: boolean;
 
+		/**
+		 * An optional flag to make the picker multi-select, if true the result is an array of picks.
+		 */
+		multiSelect?: boolean;
```

Comment:
I believe for the file open dialog we use `canSelectMany` and it would be nice to re-use the words